mmc/item_updater_mmc.cpp

```diff
@@ -141,8 +144,49 @@ bool ItemUpdaterMMC::freeSpace()
 void ItemUpdaterMMC::updateFunctionalAssociation(const std::string&)
 {}
 
+void GardResetMMC::enableDimmAndCpu()
+{
+    std::vector<std::string> intf;
+    intf.push_back("xyz.openbmc_project.Inventory.Item.Cpu");
```

Comment:
We are not allowing a processor to manually guard through the Redfish.
We are supporting DIMM and Core through the Redfish. Please use the respective core interface.